Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml

```
@@ -174,35 +187,45 @@
   - type: Sprite
-    sprite: Structures/Doors/Airlocks/Glass/cargo.rsi
+    sprite: DeltaV/Structures/Doors/Airlocks/Glass/cargo.rsi #Delta V - Resprite Doors
   - type: PaintableAirlock
     department: Logistics
   - type: Wires
     layoutId: AirlockCargo
 
+- type: entity
+  parent: AirlockGlass
+  id: AirlockHydroponicsGlass
+  suffix: Hydroponics 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)